### PR TITLE
Pedro/package installer

### PR DESCRIPTION
**Pull request by pedrocamargo**
Allows the user to install open matrix from inside QGIS, as long as they have started QGIS as administrator

### CODE REVIEW

File: matrix_procedures/display_aequilibrae_formats_dialog.py [hunk 1]
```diff
@@ -40,31 +52,51 @@ def __init__(self, iface):
...
         if self.data_type is None:
             self.error = "Path provided is not a valid dataset"
             self.exit_with_error()
-
-        self.data_type = self.data_type.upper()
-
-        if self.data_type == "AED":
-            self.data_to_show = AequilibraEData()
-        elif self.data_type == "AEM":
+        else:
```
Comment (jamiecook) [inline matrix_procedures/display_aequilibrae_formats_dialog.py:70]: does exit_with_error() not actually exit the method?
The else is superfluous (unless you've got type checking problems)

File: matrix_procedures/display_aequilibrae_formats_dialog.py [hunk 2]
```diff
@@ -40,31 +52,51 @@ def __init__(self, iface):
...
+            self.data_type = self.data_type.upper()
+
+        if self.data_type in ["AED", "AEM"]:
```
Comment (jamiecook) [inline matrix_procedures/display_aequilibrae_formats_dialog.py:73]:
```suggestion
        if self.data_type in ["AED", "AEM"]:
            load_aequilibrae_formatted_data()
        elif self.data_type == "OMX":
            load_omx_format_data()
```

You can also put in the stuff about computation_views in those methods

File: AequilibraEMenu.py [hunk 3]
```diff
@@ -315,6 +322,14 @@ def initGui(self):
             self.binary_action.triggered.connect(self.run_binary_download)
             self.binary_action.setEnabled(True)
             self.AequilibraE_menu.addAction(self.binary_action)
+
```
Comment (jamiecook) [inline AequilibraEMenu.py:325]: neat

**Review by jamiecook — APPROVED** [review]

File: matrix_procedures/display_aequilibrae_formats_dialog.py (hunk 1 above)
Comment (pedrocamargo) [reply matrix_procedures/display_aequilibrae_formats_dialog.py:70]: It actually goes through for some reason, hence the else

File: matrix_procedures/display_aequilibrae_formats_dialog.py (hunk 2 above)
Comment (pedrocamargo) [reply matrix_procedures/display_aequilibrae_formats_dialog.py:73]: I need to load the matrix first...

**Pull request merged**